### PR TITLE
Feat: Owner defaults: notif/announcement by email; Closes #1105

### LINKED ISSUE #1105: default `owner` user defaults
Should have 
Get notifications by email = True
Get announcements by email = True

### PR DESCRIPTION
**Pull request by 9921622**

### CODE REVIEW

File: src/notifications/tests/test_tasks.py [hunk 1]
```diff
@@ -41,35 +40,35 @@ def test_get_notification_emails(self):
         # No notifications to start
```
Comment (tylerecouture) [inline src/notifications/tests/test_tasks.py:40]: This comment doesn't match the assert... need to update this that the owner has turned on by default so expecting 1

File: src/announcements/tests/test_tasks.py [hunk 2]
```diff
@@ -79,7 +79,7 @@ def test_get_users_to_email(self):
                            course=course,
                            semester=semester)
         emails = get_users_to_email()
-        self.assertEqual(len(emails), 11)
+        self.assertEqual(len(emails), 12)
```
Comment (tylerecouture) [inline src/announcements/tests/test_tasks.py:82]: Add comment that 11 + owner = 12

**Review by tylerecouture — CHANGES_REQUESTED** [review]
A couple code comments is all

File: src/notifications/tests/test_tasks.py (hunk 1 above)
Comment (9921622) [reply src/notifications/tests/test_tasks.py:40]: fixed

**Comment by 9921622** [comment]
Added requested changes

**Review by tylerecouture — APPROVED** [review]

**Pull request merged**